pkgs/development/haskell-modules/all-cabal-hashes.json

```diff
@@ -0,0 +1,6 @@
+{
+    "owner": "commercialhaskell",
+    "repo": "all-cabal-hashes",
```

Comment:
One more thing here:

`all-cabal-hashes` is already in nixpkgs in `pkgs/data/misc/hackage/default.nix`.  Is there some reason to have it here as well?

Reply:
Great find! I changed the scripts to use that.